R/argument_conventions.R

```diff
@@ -1,73 +1,52 @@
-#' @param verbose logical. Should additional information be displayed to the
-#'   user. Defaults to FALSE.
-#' @param colwidths numeric vector. Column widths for use with vertical pagination.
-#' @param obj ANY. The object for the accessor to access or modify
-#' @param x An object
-#' @param \dots Passed on to methods or tabulation functions.
-#' @param value The new value
-#' @param object The object to modify in-place
-#' @param verbose logical(1). Should extra debugging messages be shown. Defaults
-#'   to \code{FALSE}.
-#' @param path character. A vector path for a position within the structure of a
-#'   `tabletree`. Each element represents a subsequent choice amongst the children
-#'   of the previous choice.
-#' @param label character(1). A label (not to be confused with the name) for the
-#'   object/structure.
-#' @param label_pos character(1). Location the variable label should be
-#'   displayed, Accepts `"hidden"` (default for non-analyze row splits), `"visible"`,
-#'   `"topleft"`, and - for analyze splits only - `"default"`.  For analyze calls,
-#'   `"default"` indicates that the variable should be visible if and only if
-#'   multiple variables are analyzed at the same level of nesting.
-#' @param cvar character(1). The variable, if any, which the content function
-#'   should accept. Defaults to NA.
-#' @param topleft character. Override values for the "top left" material to be
-#'   displayed during printing.
-#' @param page_prefix character(1). Prefix, to be appended with the split value,
-#'   when forcing pagination between the children of this split/table
-#' @param hsep character(1). Set of character(s) to be repeated as the separator
-#'   between the header and body of the table when rendered as text. Defaults to
-#'   a connected horizontal line (unicode 2014) in locals that use a UTF
-#'   charset, and to `-` elsewhere (with a once per session warning). See
-#'   [formatters::set_default_hsep()] for further information.
-#' @param indent_size numeric(1). Number of spaces to use per indent level.
-#'   Defaults to 2
-#' @param section_div character(1). String which should be repeated as a section
-#'   divider after each group defined by this split instruction, or
-#'   `NA_character_` (the default) for no section divider.
-#' @param inset numeric(1). Number of spaces to inset the table header, table
-#' body, referential footnotes, and main_footer, as compared to alignment
-#' of title, subtitle, and provenance footer. Defaults to 0 (no inset).
-#' @param table_inset numeric(1). Number of spaces to inset the table header, table
-#' body, referential footnotes, and main_footer, as compared to alignment
-#' of title, subtitle, and provenance footer. Defaults to 0 (no inset).
-#' @return NULL (this is an argument template dummy function)
-#' @rdname gen_args
+#' @name gen_args
+#' @keywords internal
 gen_args <- function(df, alt_counts_df, spl, pos, tt, tr, verbose, colwidths, obj, x,
```

Comment:
do we need this? How is it in {tern}?

Reply:
The functions ensure that all of the parameters are inherited from the other convention sections. They wouldn't be needed in `tern` since we only have one `argument_convention` section that we inherit params from there. I tested without and a lot of the documentation messed up so I think it's best to leave this as is.

Reply:
It makes sense! thank you for investigating ;)